fetch/programmer-humour/script.js

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Were these files meant to be in this pull request? The title of the pull request just says "Dog photo gallery" so I'm going to assume they're here by mistake? Can you remove them from this PR (and file a separate one if you need)?